### PR TITLE
WIP - TodoMVC es6 template replace

### PR DESCRIPTION
**Pull request by flashdesignory**
chrome before:
before_chrome
chrome after:
after_chrome
firefox before
before_firefox
firefox after:
after_firefox
safari before
before_safari
safari after
after_safari

### CODE REVIEW

File: resources/todomvc/vanilla-examples/javascript-es5/dist/index.html [hunk 1]
```diff
@@ -17,7 +24,7 @@ <h1>todos</h1>
             </header>
             <main class="main">
                 <div class="toggle-all-container">
-                    <input class="toggle-all" type="checkbox">
+                    <input class="toggle-all" type="checkbox" />
```
Comment (rniwa) [inline resources/todomvc/vanilla-examples/javascript-es5/dist/index.html:27]: This is unnecessary given this is a HTML file, not XML?

File: resources/todomvc/vanilla-examples/javascript-es6/dist/index.html [hunk 2]
```diff
@@ -17,7 +24,7 @@ <h1>todos</h1>
             </header>
             <main class="main">
                 <div class="toggle-all-container">
-                    <input class="toggle-all" type="checkbox">
+                    <input class="toggle-all" type="checkbox" />
```
Comment (rniwa) [inline resources/todomvc/vanilla-examples/javascript-es6/dist/index.html:27]: Ditto.

**Comment by rniwa** [comment]
What's the point / purpose of this change?

File: resources/todomvc/vanilla-examples/javascript-es5/dist/index.html (hunk 1 above)
Comment (flashdesignory) [reply resources/todomvc/vanilla-examples/javascript-es5/dist/index.html:27]: that happened when running format - will look into it

**Comment by flashdesignory** [comment]
> What's the point / purpose of this change?

This is referring to a slack threat in our main channel - was testing something that @julienw brought up there regarding multiple string replace calls. 
It's wip and just for testing for now

File: resources/todomvc/vanilla-examples/javascript-es6/dist/template.js [hunk 3]
```diff
@@ -47,26 +49,24 @@ class Template {
...
+                "{{list-index}}": i,
+                "{{view-index}}": i,
+            };
 
-            template = template.replace("{{id}}", data[i].id);
-            template = template.replace("{{title}}", escape(data[i].title));
-            template = template.replace("{{completed}}", completed);
-            template = template.replace("{{checked}}", checked);
-            template = template.replace("{{list-index}}", i);
-            template = template.replace("{{view-index}}", i);
```
Comment (julienw) [inline resources/todomvc/vanilla-examples/javascript-es6/dist/template.js:69]: @rniwa this is mostly to remove this, given that this makes `String.prototype.replace` very prominent for the 'add todo items' step.

**Comment by flashdesignory** [comment]
closingi this since we agreed to remove the javascript-es6 todo workload

**Pull request closed without merging**